packages/stylelint-plugin/tests/config-e2e.test.js

```diff
@@ -0,0 +1,41 @@
+const {spawnSync} = require('child_process');
```

Comment:
Copied from my learnings while wrangling e2e tests in https://github.com/prettier/stylelint-prettier